pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[project]
+name = "hipscat"
+license = {file = "LICENSE"}
+readme = "README.md"
+
+classifiers = [
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+]
+version = "0.0.1"
+
+requires-python = ">=3.7"
+dependencies = [
+    "healpy",
+    "pandas",
+    "pytest",
```

Comment:
pytest I suppose is not a real, runtime dependency, but a test optional. I would define extras e.g. `test` and `all` and maybe `docs`